From 00710956c29877e55a994701d82c92b55f6f3c93 Mon Sep 17 00:00:00 2001
From: Chen Liangjun <b36089@freescale.com>
Date: Mon, 23 Jul 2012 21:11:43 +0800
Subject: [PATCH 1830/2463] ENGR00218013-4 MX6Q ARM2: Set HDMI event as event
 2 of SDMA event

Select external SDMA request as SDMA event 2 for MX6Q ARM2 board.
SDMA event 2 can be configured HDMI or IPU.
Signed-off-by: Chen Liangjun <b36089@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_arm2.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_arm2.c b/arch/arm/mach-mx6/board-mx6q_arm2.c
index d169105..7ca4d68 100644
--- a/arch/arm/mach-mx6/board-mx6q_arm2.c
+++ b/arch/arm/mach-mx6/board-mx6q_arm2.c
@@ -1408,6 +1408,8 @@ static void hdmi_init(int ipu_id, int disp_id)
 
 	/* GPR3, bits 2-3 = HDMI_MUX_CTL */
 	mxc_iomux_set_gpr_register(3, 2, 2, hdmi_mux_setting);
+	/* GPR0, bit 0 = 1 */
+	mxc_iomux_set_gpr_register(0, 0, 1, 1);
 }
 
 /* On mx6x arm2 board i2c2 iomux with hdmi ddc,
-- 
1.7.7.4

